src/sentry/static/sentry/app/views/releaseArtifacts.jsx

```diff
@@ -27,13 +28,15 @@ var ReleaseArtifacts = React.createClass({
     this.fetchData();
   },
 
-  routeDidChange() {
-    this.fetchData();
+  componentWillReceiveProps(nextProps) {
```

Comment:
Safer to use `componentDidUpdate` since `this.state` will have old props (`nextProps` haven't propagated yet).